### PR TITLE
all functions with a comment

### PR DESCRIPTION
**Pull request by Ilansz**

### CODE REVIEW

File: js/game-logic.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ var playerTwoMoveThreeValue = undefined;
 
 function setPlayerMoves(player, moveOneType, moveOneValue, moveTwoType,
     moveTwoValue, moveThreeType, moveThreeValue) {
+//"protecting" the game from bugs
```
Comment (hugodes) [inline js/game-logic.js]: validate input

File: js/game-logic.js [hunk 2]
```diff
@@ -52,18 +53,18 @@ playerTwoMoveThreeType = moveThreeType;
 playerTwoMoveThreeValue = moveThreeValue;
 }
 }
-
+// Make sure the move type is rock, paper or scissors
```
Comment (hugodes) [inline js/game-logic.js]: validate move type

File: js/game-logic.js [hunk 3]
```diff
@@ -52,18 +53,18 @@ playerTwoMoveThreeType = moveThreeType;
...
 (moveType === 'scissors');
 }
-
+// Make sure tha the value of the move is between 1 and 99
 function isValidMoveValue(moveValue) {
 return (moveValue >= 1) && (moveValue <= 99);
 }
 
-
+// Set the winner of the round
```
Comment (hugodes) [inline js/game-logic.js]: get the winner ..

**Review by hugodes — CHANGES_REQUESTED** [review]

**Comment by Ilansz** [comment]
[Miniproject 2 bugs](https://trello.com/c/7lZghOuh/552-miniproject-2-bugs)

**Pull request merged**